.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Do we want to keep IDE configs in project or do we exclude them from repo? 
I have no particular preference on the subject, your preference would be mine! :)

Reply:
User config, no. I like to have _shareable_ config at hand for quick hands-on by peers.
If my computer fatally crash tomorrow, I may be up and running faster.

Also gives hint about the tool-belt used to work on the project. I'm not a fan of history, but this can be helpful at times 🤓